spec/lib/i18n_interpolation_fallbacks_spec.rb

```diff
@@ -32,32 +32,32 @@
       before do
         @old_locale = I18n.locale
         I18n.locale = 'it'
-        I18n.backend.store_translations('it', {"nonexistant_key" => "%{random_key} here is some Italian"})
+        I18n.backend.store_translations('it', {"nonexistent_key" => "%{random_key} here is some Italian"})
       end
       after do
         I18n.locale = @old_locale
       end
       describe "when all arguments are provided" do
         it "returns the locale's translation" do
-          expect(I18n.t('nonexistant_key', :random_key => "Hi Alex,")).to eq("Hi Alex, here is some Italian")
+          expect(I18n.t('nonexistent_key', :random_key => "Hi Alex,")).to eq("Hi Alex, here is some Italian")
         end
       end
       describe "when no arguments are provided" do
         it "returns the locale's translation without substitution" do
-          expect(I18n.t('nonexistant_key')).to eq("%{random_key} here is some Italian")
+          expect(I18n.t('nonexistent_key')).to eq("%{random_key} here is some Italian")
```

Comment:
- Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Prefer annotated tokens (like `%<foo>s`) over template tokens (like `%{foo}`).